Makefile

```diff
@@ -4,6 +4,10 @@ SOURCES := src src/common src/desktop
 ARGS    := -Ofast -flto -std=c++11 #-DDEBUG
 LIBS    := -lportaudio
 
+APPNAME      ?= NooDS

```

Comment:
Is there a reason to use `?` instead of `:`? Also, it feels a bit blargy to have both `NAME` and `APPNAME`, with the only difference being capitalization. I know the macOS build produces `NooDS.app`, but maybe we don't need a `make install` for that because manual installation is simply drag/drop anyway. For Linux, I'm fine using lowercase because that's how the binary is named as well.